operate/schema/src/main/java/io/camunda/operate/entities/VariableEntity.java

```diff
@@ -143,24 +153,22 @@ public boolean equals(Object o) {
         && Objects.equals(processDefinitionKey, that.processDefinitionKey)
         && Objects.equals(bpmnProcessId, that.bpmnProcessId)
         && Objects.equals(tenantId, that.tenantId)
-        && Arrays.equals(sortValues, that.sortValues);
+        && Objects.equals(position, that.position);
   }
 
   @Override
   public int hashCode() {
-    int result =
-        Objects.hash(
-            super.hashCode(),
-            name,
-            value,
-            fullValue,
-            isPreview,
-            scopeKey,
-            processInstanceKey,
-            processDefinitionKey,
-            bpmnProcessId,
-            tenantId);
-    result = 31 * result + Arrays.hashCode(sortValues);
```

Comment:
I would assume this calculation was there for a reason.  Do we know why and why it is not needed anymore? perhaps most this somewhere so we know in future?